static/app/views/issueDetails/traceTimeline/traceTimelineTooltip.tsx

```diff
@@ -181,3 +181,5 @@ const TraceItem = styled('div')`
   border-radius: ${p => p.theme.borderRadius};
   border-top: 1px solid ${p => p.theme.innerBorder};
 `;
+
+export {EventItem};
```

Comment:
I'm using the component as-is without any modifications.

[A local example](https://dev.getsentry.net:7999/issues/4208426984/events/abef1576630343a496798c9250fe69db/?project=1):
<img width="421" alt="image" src="https://github.com/getsentry/sentry/assets/44410/d560db68-cfe8-429f-8cc9-0de309de5880">

[Another local example](https://dev.getsentry.net:7999/issues/4912039318/events/7fef9536ef1947a9a17e9ef73c914b2b/?project=1):
<img width="454" alt="image" src="https://github.com/getsentry/sentry/assets/44410/b26fff3a-8b13-4722-98f7-362d229d63bb">